src/Styles/product_card.css

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+  background: fff;
```

Comment:
#?

Reply:
Yep, it should be `#fff`
And also it's default color, so this rule is useless here.